.travis.yml

```diff
@@ -7,12 +7,15 @@ python:
 sudo: false
 
 env:
-    - DJANGO=1.5.12
-    - DJANGO=1.6.10
+    - DJANGO=1.5.12 POSTGRES_TEST_USER=postgres POSTGRES_TEST_NAME=avocado MYSQL_TEST_USER=root MYSQL_TEST_NAME=avocado
```

Comment:
Sorry to leave such an off topic comment but is it worth testing(or even trying to support) Django 1.5? They consider [1.5 to be insecure and unsupported](https://docs.djangoproject.com/en/1.5/releases/1.5/#configurable-user-model) so it probably makes sense for Avocado to drop support as well:

![screen shot 2015-01-23 at 10 21 26 am](https://cloud.githubusercontent.com/assets/3911912/5880032/cf547ada-a2e9-11e4-9863-c14c5dd0b230.png)

This can obviously be done as a separate issue if you want but I just noticed this 1.5 entry in the travis config file and figured I'd ask.